media-sound/ebumeter-0.4.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
```

Comment:
I didn't write it. Feel free to change. I'm just passing along ebuilds I've been including in Gentoo Studio that aren't in Portage.